### ISSUE TITLE
The audio-driven video image is flickering

### ISSUE THREAD

**Issue by xiao-keeplearning** [Zejun-Yang/AniPortrait#20]
Thank you for your excellent work! The flickering in the demo video driven by audio is quite severe. Is there any solution to optimize it?

**Comment by Zejun-Yang** [comment]
Thank you for your interest! The flickering issue may be attributed to the diffusion model, and we are actively working to address this in the future.

**Comment by jkla139** [comment]
> 谢谢你的关心!闪烁的问题可能归因于扩散模式,我们正在积极努力在今后解决这一问题。

Animatediff+freenoise may can reduce flickering.

**Comment by jkla139** [comment]
FreeNoise can reduce flickering.

**Comment by Brawlence** [comment]
> Is there any solution to optimize it?
Just pass it via FFMPEG again with the time-domain median filter:

`ffmpeg.exe -i .\input.mp4 -filter:v "tmedian=3" output.mp4`

This will reduce flickering significantly

**Comment by oliverban** [comment]
> > Is there any solution to optimize it?
> 
> Just pass it through FFMPEG again, using the time-domain median filter:
> 
> `ffmpeg.exe -i .\input.mp4 -filter:v "tmedian=3" output.mp4`
> 
> This will reduce flickering significantly

Yes, but it will create glitches and artifacts like crazy! :P Cool trick for nearly static video though! 👍
